client-src/elements/form-field-specs.js

```diff
@@ -479,17 +479,48 @@ export const ALL_FIELDS = {
     type: 'textarea',
     attrs: {rows: 2},
     required: false,
-    label: 'TAG Review',
-    help_text: html`Link(s) to TAG review(s), or explanation why this is
-                not needed.`,
+    label: 'TAG Specification Review',
+    help_text: html`
+      Link(s) to TAG specification review(s), or explanation why this is not needed.`,
+    extra_help: html`
+    <p>
+      The <a target="_blank" href="https://www.w3.org/2001/tag/">W3C Technical Architecture Group</a> (TAG)
+      is a special working group of the W3C that consists of a few appointed and elected members,
+      all of whom are experienced members of the web standards community.
+      The Blink launch process has a formal requirement for requesting a
+      <a target="_blank" href="https://github.com/w3ctag/design-reviews">TAG specification review</a>
+      for all features. The review happens publicly on a GitHub issue.
+    </p>
+    <p>
+      You will likely have asked for an "<a target="_blank" href=
+        "https://github.com/w3ctag/design-reviews/issues/new?template=005-early-design-review.md"
+      >Early Design Review</a>" earlier in the process to get the TAG familiar with your feature.
+      This isn't that.
+    </p>
+    <p>
+      It's recommended that you file a TAG <a target="_blank" href=
+        "https://github.com/w3ctag/design-reviews/issues/new?template=010-specification-review.md"
+      >Specification Review</a> as soon as your specification is written, during your Origin Trial.
```

Comment:
Note that the Origin Trials stage won't apply to all new features with specs. We could possibly say "It's recommended that you file a TAG Specification Review as soon as your specification is written, and at least a month ahead of sending an Intent to Ship."

Or similar. But it's probably fine as-is (but might confuse some folks. 🤷)

Reply:
Updated with your suggestions:
```
It's recommended that you file a TAG Specification Review as soon as your specification is written, 
and at least a month ahead of sending an Intent to Ship.  
There might be enough time if you file for review of your specification during your Origin Trial, assuming your feature requires one.
```

Maybe this is a bit wordy and unnecessary.

Reply:
removed the Origin Trial part.